fundamentals/trees/test/trees.test.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+const jsdom = require('jsdom');
+const { JSDOM } = jsdom;
+const fs = require('fs');
+const path = require('path');
```

Comment:
`use strict` is presumed in ES6. Also, we should leave a comment about these (they haven't _really_ seen modules yet.